### PR TITLE
OCPBUGS-45802: fix Number input focus layout issue

### PR DESCRIPTION
**Pull request by rhamilto**
After

Screenshot 2024-12-06 at 9 29 10 AM

### CODE REVIEW

File: frontend/public/style/ancillary/_patternfly4.scss [hunk 1]
```diff
@@ -95,7 +95,6 @@
   --pf-v5-c-form-control--FontSize: var(--pf-v5-global--FontSize--md);
   --pf-v5-c-form-control--LineHeight: var(--pf-v5-global--LineHeight--md);
   --pf-v5-c-form-control--Resize: none;
-  --pf-v5-c-form-control--OutlineOffset: 0;
```
Comment (rhamilto) [inline frontend/public/style/ancillary/_patternfly4.scss:98]: This is the bug fix.  The rest of the changes are just linting fixes.

**Review by sg00dwin — APPROVED** [review]
/lgtm

**Pull request merged**